EventSubscriber/EventSubscriberInterface.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\EventSubscriber;
+
+interface EventSubscriberInterface
```

Comment:
Shouldn't it extend another interface? That would make it easier on the user.

Reply:
Good idea. Fixed.